packages/errors/src/errors/CustomErrorBase.ts

```diff
@@ -41,7 +41,7 @@ export class CustomErrorBase extends Error {
   constructor(message?: string, cause?: Error | unknown) {
     let fullMessage = message;
     if (cause !== undefined) {
-      const causeStr = stringifyError(cause);
+      const causeStr = stringifyError(cause).substring(0, 5000);
```

Comment:
Don't really want to do this in this base class, it's used in all kind of different places. Would it be problematic to skip this change and just keep the other one?

Reply:
That should be fine. The large majority of the errors were coming from `UrlReaderProcessor` anyway, so that should be enough to avoid the crash. I've pushed the change.